hw2/05-form.html

```diff
@@ -7,14 +7,167 @@
+          />
+        </div>
+        <div class="mb-3">
+          <label for="email" class="form-label fw-bold">Email:</label>
+          <input
+            type="email"
+            class="form-control"
+            id="email"
+            name="Email"
+            placeholder="Enter your email"
+            required
+          />
+        </div>
+        <div class="mb-3">
+          <label for="registrationStatus" class="form-label fw-bold"
+            >Registration Status:</label
+          >
+          <select
+            class="form-select"
+            name="Registration Status"
+            id="registrationStatus"
+            required
+          >
+            <option selected disabled value="">Choose an Option</option>
+            <option value="new">New</option>
+            <option value="returning">Returning</option>
+            <option value="guest">Guest</option>
+          </select>
+        </div>
+        <fieldset class="mb-3">
+          <legend class="fw-bold" style="font-size: 1rem">
+            Have you taken any of the following courses?
+          </legend>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="programming"
+              id="courseProgramming"
+              name="courseProgramming"
+            />
+            <label class="form-check-label" for="courseProgramming">
+              Programming Languages
+            </label>
+          </div>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="operating"
+              id="courseOperating"
+              name="courseOperating"
+            />
+            <label class="form-check-label" for="courseOperating">
+              Operating Systems
+            </label>
+          </div>
+          <div class="form-check">
+            <input
+              class="form-check-input"
+              type="checkbox"
+              value="fullstack"
+              id="courseFullStack"
+              name="courseFullStack"
+            />
+            <label class="form-check-label" for="courseFullStack">
+              Full Stack Web Development
+            </label>
+          </div>
+        </fieldset>
+        <div class="mb-3">
+          <label for="additionalInfo" class="form-label fw-bold"
+            >Anything else?</label
+          >
+          <textarea
+            class="form-control"
+            id="additionalInfo"
+            name="Additional Info"
+            rows="3"
+            required
+          ></textarea>
+        </div>
+        <div class="d-flex w-100 mb-3">
+          <button type="submit" class="btn btn-primary w-100 me-1">
+            Submit
+          </button>
+          <button type="reset" class="btn btn-secondary w-100 ms-1">
+            Reset
+          </button>
+        </div>
       </form>
     </main>
+
```

Comment:
Consider adding a comment here that says <--Modal--> just to make it a little bit more clear that that is where the modal is.

Reply:
Added a comment in form.html indicating where the modal code begins.